ztrouy/frontend-capstone-api
# Api Initialization

### Purpose
To provide the base for a proper web API for Game Gauge built in Python and Django, enabling more robust and complex features to be added in the future. It also enables greater security features to be used throughout the site.

### To Test
Fetch branch, run `pipenv shell`, then `./seed_database.sh`, and confirm the following
- All fixtures are loaded correctly, providing Users, Tokens, Groups, User Groups, Games, User Games, Genres, Game Genres, Platforms, and Game Platforms
- Database is created as `db.sqlite3`

closes #3 
closes #4
